R/ard_wilcoxtest.R

```diff
@@ -0,0 +1,201 @@
+#' ARD Wilcoxon Rank-Sum Test
+#'
+#' @description
+#' Analysis results data for paired and non-paired Wilcoxon Rank-Sum tests.
+#'
+#' @param data (`data.frame`)\cr
+#'   a data frame. See below for details.
+#' @param by ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   column name to compare by
+#' @param variable ([`tidy-select`][dplyr::dplyr_tidy_select])\cr
+#'   column name to be compared
```

Comment:
```suggestion
#'   column name to be compared.
```
All of this is not really relevant but why not